esmero/archipelago-aws-demo
# Composer updates to Drupal 8.6.14 and some candy

This pull updates core packages, adds new contributed modules and also provides the first 3 seeds for our strawberry seeds module. 

# HOW TO TEST?

```bash
composer install
```

that will deal with using the lock file for updating packages and bringing your dependencies into shape

For the last part, this can be tested (if you are in rush) by follwing this rough guide: (please wait for full docs if you fear this is just too little, some updated configs are coming too)

This requires JSONAPI to be enabled (make sure you do this in /admin/extend) and POST needs to be enabled too (admin/config/jsonapi)

You need a jsonapi user with password (you choose!) and role "Metadata Pro"

For now go in this order:
```SHELL
curl --user jsonapi:yourchosenpassword -H 'Accept: application/vnd.api+json' -H 'Content-type: application/vnd.api+json' -XPOST http://localhost:8001/jsonapi/metadatadisplay_entity/metadatadisplay_entity --data-binary @d8content/metadatadisplay_entity_01.json

curl --user jsonapi:yourchosenpassword -H 'Accept: application/vnd.api+json' -H 'Content-type: application/vnd.api+json' -XPOST http://localhost:8001/jsonapi/metadatadisplay_entity/metadatadisplay_entity --data-binary @d8content/metadatadisplay_entity_02.json

curl --user jsonapi:yourchosenpassword -H 'Accept: application/vnd.api+json' -H 'Content-type: application/vnd.api+json' -XPOST http://localhost:8001/jsonapi/metadatadisplay_entity/metadatadisplay_entity --data-binary @d8content/metadatadisplay_entity_03.json
````

This adds our basic metadata displays to your demo instance. This will be automatized so if you are reading this its because you are totally aware of what you are doing! =)

@giancarlobi just letting you know. jsonapi is a thing now for us =)
